File: backend/CoreCuestionariosOIJ/src/CuestionariosEntidades/Models/AnswerOption.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace CuestionariosEntidades.Models
{
    public class AnswerOption
    {
        public int? IdAnswer { get; set; }
        public int? IdOption { get; set; }
        public bool? IsDeleted { get; set; }

        // EF Relation
        public Option? Option { get; set; }
        public Answer? Answer { get; set; }
    }
}

File: frontend/src/app/models/AnswerOption.ts
import { Answer } from "./Answer";
import { Option } from "./Option"

export class AnswerOption{
  
    idAnswer?: number
    idOption?: number
    option?: Option;
    answer?: Answer;
    
    constructor({idAnswer, idOption}:{idAnswer?: number, idOption?: number}) {
        this.idAnswer = idAnswer;
        this.idOption = idOption;
    }

}